### ISSUE TITLE
install previous version of xgboost, to avoid weird scipy.sparse issues

### ISSUE THREAD

**Issue by ClimbsRocks** [ClimbsRocks/auto_ml#53]
or, look into installing sklearn's gradient booster

**Comment by ClimbsRocks** [comment]
worked around this in a different way. also using sklearn's gradient booster, and that works pretty well (not as well as xgboost, but better than most other algorithms).
